examples/everything.amp.html

```diff
@@ -196,8 +196,8 @@ <h1 id="top">AMP #0</h1>
   </p>
 
   <amp-lightbox id="lightbox1" class="lightbox1" layout="nodisplay">
-    <div class="lightbox1-content">
-      <amp-img id="img0" src="https://lh4.googleusercontent.com/-okOlNNHeoOc/VbYyrlFYFII/AAAAAAABYdA/La-3j3c-QQI/w452-h900-p/PANO_20150726_171347%257E2.jpg" width=226 height=450 layout="responsive"></amp-img>
+    <div class="lightbox1-content" close>
+      <amp-img id="img0" src="https://lh4.googleusercontent.com/-okOlNNHeoOc/VbYyrlFYFII/AAAAAAABYdA/La-3j3c-QQI/w452-h900-p/PANO_20150726_171347%257E2.jpg" width=226 height=450 layout="responsive" on="tap:lightbox1.close"></amp-img>
```

Comment:
The spec (md file) says close attr should work , hence the support for the close attr.